src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -217,6 +231,13 @@ private IEnumerable<IConfigurationSection> GetFeatureDefinitionSections()
                 return featureManagementConfigurationSection.GetChildren();
             }
 
+            //
+            // There is no "FeatureManagement" section in the configuration
```

Comment:
we shouldn't be performing the check for the featuremanagement section if this flag is set.